install/updateVersion.sh

```diff
@@ -55,8 +55,11 @@ function gen_file() {
     fi
 }
 
+TARGETS="istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml"
```

Comment:
in go, you don't need to split lines.

Reply:
I split the lines for readability. Alternatively, it could be written with line continuations (see below), but this seemed less error prone. Also, this is a bash script :)

```bash
TARGETS="istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml \
    istio-multicluster.yaml istio-auth-multicluster.yaml istio-remote.yaml \
    istio-galley.yaml istio-auth-galley.yaml"
```